### ISSUE TITLE
Request: include more logic in the library so it's easier to use

### ISSUE THREAD

**Issue by jpage4500** [alexeyvasilyev/rtsp-client-android#15]
Hi, this is just a request for the future.. I want to use this library in my app to support RTSP video playback. But, there's a LOT of logic in your test app that's not part of the library - VideoDecodeThread, AudioDecodeThread, etc.

It'd be really nice if this was part of the library and I could just pass in a SurfaceView and some other details - like URL/user/pass/port and let the library handle the rest. Maybe there's plenty of reasons this wouldn't make sense - but it seems like it'd make this library a lot more appealing to integrate without having to learn a lot of details.

Anyway, I'm trying now to port over the demo app's classes to my app and I'm able to get the video stream working - but once it's stopped it doesn't reconnect. I'm sure it's something I'm missing but just wanted to mention the thought about including more logic into the library too

thanks
joe

**Comment by alexeyvasilyev-wyze** [comment]
I'm planning to add RtspSurfaceView soon.

**Comment by alexeyvasilyev** [comment]
RtspSurfaceView implemented in https://github.com/alexeyvasilyev/rtsp-client-android/commit/ba9a7bcc17335e4b06b60219e2401ff59a3d4dc7

This is how to use it:
```
 <com.alexvas.rtsp.widget.RtspSurfaceView
       android:layout_width="match_parent"
       android:layout_height="match_parent"
       android:id="@+id/svVideo" />
```

```
val uri = Uri.parse("rtsp://192.168.1.3/videoMain")
binding.svVideo.init(uri, username, password)
binding.svVideo.start(true, true)

```

**Comment by alexeyvasilyev** [comment]
Fixed in 2.0.0

**Comment by jpage4500** [comment]
perfect - thanks!
